app/src/main/java/com/imashnake/animite/features/home/MediaSmallRow.kt

```diff
@@ -20,7 +20,7 @@ fun MediaSmallRow(mediaList: List<MediaListQuery.Medium?>, onItemClick: (itemId:
     ) {
         items(mediaList) { media ->
             MediaSmall(
-                image = media?.coverImage?.large,
+                image = media?.coverImage?.extraLarge,
                 anime = media?.title?.romaji ?:
```

Comment:
Tempted to say a delegate property inside the Title would be cleaner to not have to do this `title?.romanji ?: title?.english ?: title?.native` on a single line in a Composable

Reply:
Delegate how? These classes are generated by `apollo-kotlin` btw.

Reply:
I might move these checks to the `ViewModel` like for `MediaPage`, so I'll mark this as resolved for now.